packages/manager/apps/pci-public-ip/src/components/list/FloatingIP.tsx

```diff
@@ -0,0 +1,127 @@
+import {
+  DataGridTextCell,
+  Datagrid,
+  useDatagridSearchParams,
+} from '@ovhcloud/manager-components';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import {
+  ODS_BUTTON_SIZE,
+  ODS_BUTTON_VARIANT,
+  ODS_ICON_NAME,
+  ODS_ICON_SIZE,
+  ODS_MESSAGE_TYPE,
+  ODS_SPINNER_SIZE,
+} from '@ovhcloud/ods-components';
+import {
+  OsdsButton,
+  OsdsIcon,
+  OsdsLink,
+  OsdsMessage,
+  OsdsSpinner,
+} from '@ovhcloud/ods-components/react';
+import { useTranslation } from 'react-i18next';
+import { FloatingIP } from '@/interface';
+import { useFLoatingIPs } from '@/hooks/useFloatingIP';
+import Actions from './Actions';
+
+export default function FloatingIPComponent({ projectId, projectUrl }) {
+  const { t } = useTranslation('common');
+
+  const { pagination, setPagination } = useDatagridSearchParams();
+
+  const { error, data: floatingIPs, isLoading } = useFLoatingIPs(
+    projectId || '',
+    { pagination },
+  );
+
+  const goToInstanceHref = (id: string) => `${projectUrl}/instances/${id}`;
```

Comment:
for url pointing to other apps you can use navigation plugin from the shell to create URLs (passing application id: 'public-cloud')

Reply:
the projectUrl is coming from the shell actually, it's declared in the parent component ListingPage
```
  useEffect(() => {
    navigation
      .getURL('public-cloud', `#/pci/projects/${projectId}`, {})
      .then((data) => {
        setProjectUrl(data as string);
      });
  }, [projectId, navigation]);
```

Or maybe i didn't understood well your suggestion ?

Reply:
ok if you are passing the URL this way it should be good